pkgs/development/libraries/grantlee/5/setup-hook.sh

```diff
@@ -4,20 +4,14 @@ providesGrantleeRuntime() {
     [ -d "$1/$grantleePluginPrefix" ]
 }
 
-_grantleeCrossEnvHook() {
+_grantleeEnvHook() {
     if providesQtRuntime "$1"; then
         propagatedBuildInputs+=" $1"
         propagatedUserEnvPkgs+=" $1"
     fi
 }
-crossEnvHooks+=(_grantleeCrossEnvHook)
-
-_grantleeEnvHook() {
-    if providesGrantleeRuntime "$1"; then
```

Comment:
Is it valid to use `providesQtRuntime` instead of `providesGrantleeRuntime` in the native case?

Reply:
Ooo I bet it should have been grantlee for both all along.